### ISSUE TITLE
`:set tabstop=7` command affects all buffers and is even saved to settings.json

### ISSUE THREAD

**Issue by Chiel92** [VSCodeVim/Vim#6095]
**To Reproduce**

1.  Run `:set tabstop=7`
2. Observe the changed settings.json

**Expected behavior**
The tabstop/tabsize setting should only have been changed for the current buffer/document.
If I wanted a global change I would have edited the settings.json myself.

- Extension (VsCodeVim) version: 1.18.8
- VSCode version: 1.53 insiders
- OS: windows 10

**Comment by J-Fields** [comment]
Right, the issue here is basically that we don't currently differentiate between global and local options (see [:help local-options](https://vimhelp.org/options.txt.html#local-options))

**Comment by Chiel92** [comment]
Is this a VSCode limitation, or a VSCodeVim limitation?

**Comment by J-Fields** [comment]
At least in this instance, a VSCodeVim limitation
